samples/serial/serial-over-network.md

```diff
@@ -0,0 +1,228 @@
+# Connecting to serial port over network
+
+Suppose in a workflow, you have an application that works with some devices attached to serial ports. When moving the workflow to run on the EFLOW VM, you have problems accessing serial ports since the EFLOW VM is isolated from serial ports attached to the Windows host. This article describes how to configure the EFLOW VM and host to redirect communications to a serial port over the network. With the redirection, applications running on EFLOW VM can communicate with devices attached to serial ports (USB serial ports included) on the host.
```

Comment:
When moving the workflow to run on the EFLOW VM, you **may** have problems accessing serial ports since the EFLOW VM is isolated from serial ports attached to the Windows host. 

This article describes how to configure the EFLOW VM and host to redirect communications to a serial port over the network. With the redirection, applications running on EFLOW VM can communicate with devices attached to serial ports (USB serial ports included) on the host.

Reply:
add the word 'may' (no need to bold) and add spacing between these two statements

Reply:
updated.